projects/angular/src/progress/spinner/_spinner.clarity.scss

```
@@ -13,58 +13,186 @@
     // spinner-lg is the default
     position: relative;
     display: inline-block;
-    @include mixins.min-equilateral(variables.$clr_baselineRem_3);
+    @include mixins.min-equilateral-var(
+      clr-spinner-default-size,
+      variables.$clr_baselineRem_3,
+      variables.$clr-use-custom-properties
+    );
     animation: spin 1s linear infinite;
-    margin: 0;
+    margin-right: 0;
     padding: 0;
-    background: icons.generateSpinnerIcon(
-      spinner-variables.$clr-spinner-bg-color,
-      spinner-variables.$clr-spinner-color,
-      spinner-variables.$clr-spinner-opacity
-    );
     text-indent: 100%;
     overflow: hidden;
     white-space: nowrap;
 
+    &::after {
+      content: '';
+      @include mixins.min-equilateral-var(
+        clr-spinner-default-size,
+        variables.$clr_baselineRem_3,
+        variables.$clr-use-custom-properties
+      );
+      position: absolute;
+      top: 0;
+      left: 0;
+      clip-path: ellipse(100% 53.5% at 0 0);
+      @include mixins.css-var(
+        border-width,
+        clr-spinner-default-stroke-width,
+        variables.$clr_baselineRem_0_25,
+        variables.$clr-use-custom-properties
+      );
+      border-style: solid;
+      @include mixins.css-var(
+        border-color,
+        clr-spinner-fill-snake-color,
+        spinner-variables.$clr-spinner-color,
+        variables.$clr-use-custom-properties
+      );
+      border-right-color: transparent !important;
+      border-bottom-color: transparent !important;
+      border-radius: 50%;
+    }
+
+    &::before {
+      content: '';
+      @include mixins.min-equilateral-var(
+        clr-spinner-default-size,
+        variables.$clr_baselineRem_3,
+        variables.$clr-use-custom-properties
+      );
+      position: absolute;
+      top: 0;
+      left: 0;
+      @include mixins.css-var(
+        border-width,
+        clr-spinner-default-stroke-width,
+        variables.$clr_baselineRem_0_25,
+        variables.$clr-use-custom-properties
+      );
+      border-style: solid;
+      @include mixins.css-var(
+        border-color,
+        clr-spinner-fill-bg-color,
+        rgba(spinner-variables.$clr-spinner-bg-color, spinner-variables.$clr-spinner-opacity),
+        variables.$clr-use-custom-properties
+      );
+      border-radius: 50%;
```

This should be extracted as a property token. Same as previous comment.
```suggestion
      @include mixins.css-var(
        border-radius,
        clr-spinner-radius,
        50%,
        variables.$clr-use-custom-properties
      );
```
Add it in properties and shim as:
Prop: `--clr-spinner-radius: 50%;` 
Shim: `--clr-spinner-radius: var(--cds-alias-object-border-radius-300);`